contracts/Enigma.sol

```diff
@@ -191,7 +260,7 @@ contract Enigma is EnigmaStorage, EnigmaEvents, Getters, Ownable {
     function deposit(address _custodian, uint _amount)
     public
     isUpdatedEnigmaContract
-    workerRegistered(_custodian)
+    workerRegistered(state.stakingToOperatingAddresses[_custodian])
```

Comment:
Quick question - does this mean that the setup flow must be `setOperatingAddress` -> `register` -> `deposit`?

Reply:
Yes

Reply:
Ah, sorry my bad. @apalepu23 points out to me that the correct flow is:
`register` (by operating address) -> `setOperatingAddress` (by staking address) -> `deposit` (by staking address)